rmgpy/data/kinetics/database.py

```diff
@@ -799,3 +802,71 @@ def reconstructKineticsFromSource(self, reaction, source, fixBarrierHeight=False
                 if isinstance(kinetics, ArrheniusEP):
                     kinetics = kinetics.toArrhenius(H298)
                 return kinetics
+
+    def loadFilterFits(self, database_directory, rxn_systems):
+        """
+        Load Arrhenius fits of the highest rates in each training reaction family from a file located at `fits_location`
+        on disk.
+
+        For reaction filtering, the loaded fits per reaction family are sorted to correspond to the user
+        defined families.
+        """
+
+        families = self.families.keys()
+
+        fits_location = os.path.join(database_directory, 'FilterArrheniusFits.yml')
+
+        logging.debug("""Loading Arrhenius fits from {0} based on training reactions to generate custom filter 
+        criteria for each reaction family""".format(os.path.join(fits_location)))
+
+        class_dictionary = {'ScalarQuantity': ScalarQuantity,
+                            'Arrhenius': Arrhenius,
+                            'FilterLimitFits': FilterLimitFits,
+                            }
+
+        obj = FilterLimitFits()
+        obj.load_yaml(fits_location, class_dictionary)
+
+        unimol_kinetics_list = []
+        bimol_kinetics_list = []
+        for family in families:
+            for filter_family, fit in obj.unimol.items():
+                if family == filter_family:
+                    unimol_kinetics_list.append(fit)
+                    break
+            else:  # We could not find the filter fits for this family. Raise an exception or else the indices will be off
+                raise ValueError('Unable to find filter fits for family {0}'.format(family))
```

Comment:
Perhaps include in the message what the user should do? e.g. fit filter thresholds for that family

Aside: I'm not super keen on `fits` as a noun. It just sounds weird...